cvr/src/main/java/org/cbioportal/cmo/pipelines/cvr/sv/GMLSvDataReader.java

```diff
@@ -108,7 +109,12 @@ private void processJsonFile() {
             if (samples != null) {
                 for (GMLCnvIntragenicVariant cnv : result.getCnvIntragenicVariantsGml()) {
                     for (String sampleId : samples) {
-                        CVRSvRecord record = new CVRSvRecord(cnv, sampleId);
+                        CVRSvRecord record = svUtilities.makeCvrSvRecordFromGmlCnvIntragenicVariant(cnv, sampleId);
+                        try {
+                            svUtilities.populateEventInfoFromGmlCnvIntragenicVariant(record);
```

Comment:
considering we are already making the CVRSvRecord from CnvIntrangenic, couldn't we call the populateeventinfo method from inside that function? seems like an extra step when it's all setting data members within the same class